### PR TITLE
[Multi-Database Support] Optimize table case sensitivity

### PR DESCRIPTION
**Pull request by Shoothzj**
## What's the purpose of this PR
I think if we want to support more databases, we should be more rigorous and pay closer attention to case sensitivity in more places.

## Brief changelog
Quoted table name.

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Read the [Contributing Guide](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit tests to verify the code.
- [x] Run `mvn clean test` to make sure this pull request doesn't break anything.
- [x] Update the [`CHANGES` log](https://github.com/apolloconfig/apollo/blob/master/CHANGES.md).

### CODE REVIEW

File: apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/CommitRepository.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ List<Commit> findByAppIdAndClusterNameAndNamespaceNameAndDataChangeLastModifiedT
       String appId, String clusterName, String namespaceName, Date dataChangeLastModifiedTime, Pageable pageable);
 
   @Modifying
-  @Query("update Commit set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where appId=?1 and clusterName=?2 and namespaceName = ?3 and isDeleted = false")
+  @Query("update Commit set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where appId=?1 and clusterName=?2 and namespaceName = ?3 and IsDeleted = false")
```
Comment (nobodyiam) [inline apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/CommitRepository.java:38]: Is it necessary to change appId, clusterName and namespaceName to AppId, ClusterName and NamespaceName?
```suggestion
  @Query("update Commit set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where AppId=?1 and ClusterName=?2 and NamespaceName = ?3 and IsDeleted = false")
```

File: apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ItemRepository.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ public interface ItemRepository extends PagingAndSortingRepository<Item, Long> {
   Item findFirst1ByNamespaceIdOrderByLineNumDesc(Long namespaceId);
 
   @Modifying
-  @Query("update Item set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?2 where namespaceId = ?1 and isDeleted = false")
+  @Query("update Item set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?2 where namespaceId = ?1 and IsDeleted = false")
```
Comment (nobodyiam) [inline apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ItemRepository.java:47]: Is it necessary to change namespaceId to NamespaceId?
```suggestion
  @Query("update Item set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?2 where NamespaceId = ?1 and IsDeleted = false")
```

File: apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/NamespaceRepository.java [hunk 3]
```diff
@@ -33,7 +33,7 @@ public interface NamespaceRepository extends PagingAndSortingRepository<Namespac
   Namespace findByAppIdAndClusterNameAndNamespaceName(String appId, String clusterName, String namespaceName);
 
   @Modifying
-  @Query("update Namespace set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?3 where appId=?1 and clusterName=?2 and isDeleted = false")
+  @Query("update Namespace set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?3 where appId=?1 and clusterName=?2 and IsDeleted = false")
```
Comment (nobodyiam) [inline apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/NamespaceRepository.java:36]: The same as above.

File: apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ReleaseHistoryRepository.java [hunk 4]
```diff
@@ -39,7 +39,7 @@ Page<ReleaseHistory> findByAppIdAndClusterNameAndNamespaceNameOrderByIdDesc(Stri
   Page<ReleaseHistory> findByReleaseIdAndOperationInOrderByIdDesc(long releaseId, Set<Integer> operations, Pageable pageable);
 
   @Modifying
-  @Query("update ReleaseHistory set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where appId=?1 and clusterName=?2 and namespaceName = ?3 and isDeleted = false")
+  @Query("update ReleaseHistory set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where appId=?1 and clusterName=?2 and namespaceName = ?3 and IsDeleted = false")
```
Comment (nobodyiam) [inline apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ReleaseHistoryRepository.java:42]: The same as above.

File: apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ReleaseRepository.java [hunk 5]
```diff
@@ -48,7 +48,7 @@ Release findFirstByAppIdAndClusterNameAndNamespaceNameAndIsAbandonedFalseOrderBy
   List<Release> findByIdIn(Set<Long> releaseIds);
 
   @Modifying
-  @Query("update Release set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where appId=?1 and clusterName=?2 and namespaceName = ?3 and isDeleted = false")
+  @Query("update Release set IsDeleted = true, DeletedAt = ROUND(UNIX_TIMESTAMP(NOW(4))*1000), DataChange_LastModifiedBy = ?4 where appId=?1 and clusterName=?2 and namespaceName = ?3 and IsDeleted = false")
```
Comment (nobodyiam) [inline apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ReleaseRepository.java:51]: The same as above.

File: apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ItemRepository.java (hunk 2 above)
Comment (Shoothzj) [reply apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/ItemRepository.java]: @nobodyiam Thanks, fixed

File: apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/CommitRepository.java (hunk 1 above)
Comment (Shoothzj) [reply apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/repository/CommitRepository.java]: @nobodyiam Thanks, fixed

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**